sale_commission/wizard/wizard_invoice.py

```diff
@@ -13,6 +13,10 @@ def _default_journal(self):
         return self.env['account.journal'].search(
             [('type', '=', 'purchase')])[0]
 
+    def _default_refund_journal(self):
+        return self.env['account.journal'].search(
+            [('type', '=', 'purchase_refund')])[0]
```

Comment:
You can put `[:1]` instead of `[0]` to avoid an error in case there's no journal on the company (also for the other journal).